sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityAsyncTests.java

```diff
@@ -285,15 +335,16 @@ public void issueTokenWithResponse(HttpClient httpClient) {
         StepVerifier.create(
             asyncClient.createUser()
                 .flatMap(communicationUser -> {
-                    List<String> scopes = new ArrayList<>(Arrays.asList("chat"));
+                    List<CommunicationTokenScope> scopes = 
+                        new ArrayList<>(Arrays.asList(CommunicationTokenScope.CHAT));
                     return asyncClient.issueTokenWithResponse(communicationUser, scopes);
                 }))
             .assertNext(issuedToken -> {
                 assertNotNull(issuedToken.getValue().getToken());
                 assertFalse(issuedToken.getValue().getToken().isEmpty());
-                assertNotNull(issuedToken.getValue().getExpiresOn());
-                assertFalse(issuedToken.getValue().getExpiresOn().toString().isEmpty());
-                assertNotNull(issuedToken.getValue().getUser());
```

Comment:
keep this line

Reply:
the issuedToken is updated to type AccessToken and no longer contains the user.